Pojav Launcher instructions (#56)
Fabulously-Optimized/wiki: en-us/install-instructions.md

A simple installer for vanilla launcher [is coming soon](https://github.com/Fabulously-Optimized/fabulously-optimized/issues/110)!

### Pojav Launcher (Android)

> **This process is lengthy, may not work or may not improve performance for you. You've been warned.**

Reviewer: ```suggestion
**This process is lengthy, it may not work or may not improve performance for you. You've been warned.**
```
Is it worth saying that you may or may not receive support using this launcher? Since there's no PR for main repo, it won't be selectable in a bug report (ofc you could make one, but that's besides the point)

> All the following instructions should be ran on your high-end Android device.

Reviewer: ```suggestion
All the following instructions should be run on your high-end Android device.
```

1. [Download Pojav Launcher](https://play.google.com/store/apps/details?id=net.kdt.pojavlaunch)
   * Do not fall into scam websites. The only official site for it is https://github.com/PojavLauncherTeam/PojavLauncher
2. [Download Material Files](https://play.google.com/store/apps/details?id=me.zhanghai.android.files)
   * This is the file manager we'll be using for this tutorial. You could use others too, but instructions may vary.

> 3. [Download Fabric installer](https://fabricmc.net/use/installer/) (jar version)

Reviewer: ```suggestion
3. [Download the Fabric installer](https://fabricmc.net/use/installer/) (as a `.jar` file)
```

Author: well there is an exe version and a jar version... 
not a "list of files"

4. [Download the latest **MultiMC version** of Fabulously Optimized](https://www.curseforge.com/minecraft/modpacks/fabulously-optimized/files)
   * Don't see the MultiMC version? Click the title of the CurseForge version you need and scroll down, you'll find the MultiMC variant there.
      * Alternatively, look for it in the sidebar, under "server packs". Don't worry, these are not server packs, just marked as such for findability.
   * If you don't see any mods inside the zip or only see one, you downloaded the wrong version.
5. [Download Better Sodium Video Settings Button](https://modrinth.com/mod/better-sdoium-video-settings-button/versions)
   * This is a workaround because the original video settings don't work on mobile for some reason.
6. Run Pojav Launcher
7. Tap `Add account` -> `Microsoft account` and sign in
8. Tap the settings button -> `Video and renderer`-> `ANGLE (release 1.17+, mid)` -> `Back to the last screen`-> `Back to the last screen`
9. Tap `Create new profile` -> `Create new profile` -> `Save profile`
10. Tap `Play`. Wait for it to fully load to the title screen, then exit the game.

>     * If you can't get past a black screen, try creating a new 1.18.2 profile first - I think it downloads the necessary Java and dependencies.

Author: you have a typo on your suggestion too lol

>     * If you can't get past a black screen, try creating a new 1.18.2 profile first - I think it downloads the necessary Java and dependencies.

Reviewer: :ioa:

11. Launch Pojav again, tap `Install .jar` -> select `fabric-installer-x.x.x.jar`. You'll see the Fabric installer with some logs.
12. Tap :x: to close the logs, then `Install` -> `OK` -> `[X]`
13. Close Pojav and open Material Files.
14. Navigate to `Download` -> `Fabulously+Optimized+x.x.x.zip` -> `Fabulously Optimized x.x.x` -> `minecraft`
15. Hold down on one folder and tap on all three folders
16. Tap the `Extract` button at the top

> 17. Navigate back to root, then `Android` -> `data` -> `net.kdt.pojavlaunch` -> `files` -> `.minecraft`

Reviewer: ```suggestion
17. Navigate back to the root directory, then `Android` -> `data` -> `net.kdt.pojavlaunch` -> `files` -> `.minecraft`
```

    * If it's easier for you, press 3 dots -> `Go to` -> paste `/storage/emulated/0/Android/data/net.kdt.pojavlaunch/files/.minecraft` -> `OK`
18. Tap `Paste` at the bottom, select `Apply this action to all files`, tap `Merge`
19. Navigate to `Download`, hold down on `bsvsb-x.x.x.jar`, select `Cut` from the top
20. Navigate back to root, then `Android` -> `data` -> `net.kdt.pojavlaunch` -> `files` -> `.minecraft` -> `mods`
    * If it's easier for you, press 3 dots -> `Go to` -> paste `/storage/emulated/0/Android/data/net.kdt.pojavlaunch/files/.minecraft/mods` -> `OK`
21. Tap `Paste`, close Material Files.
22. Launch Pojav, make sure `fabric-loader-x.x.x - fabric-loader-x.x.x-x.x.x` is selected, `Play`.
23. Fabulously Optimized should now be running! 

> Note that FO's options.txt does not apply as Pojav overwrites it. You may want to change your GUI scale to 6 for better usage.

Author: Note to self: if I take Pojav seriously, I should create an options.txt for it and include it in these instructions.

* This process is for Android devices - phones and tablets. iOS devices _could_ work, but would also require even more steps and is untested.
* Your device must be "mid-range" or "high-end" if you want decent FPS. 6+ GB RAM recommended.
* Pojav Launcher supports Android 5 and up, but these instructions were tested on Android 12L, so some things may vary.
* Your device will likely heat and consume a lot of battery as you play. It is not as device-optimized as Bedrock Edition.
* Attempts have been made to improve performance, but depending on your device you may need to adjust video settings.
* In the worst case, you may not get it running at all. Try again after a Pojav Launcher update.

<details><summary><a href="https://play.google.com/store/apps/details?id=com.mojang.minecraftpe">I don't agree and just want to play Minecraft</a></summary></details>

<details>
  <summary>I agree and would like to see the instructions</summary>

1. [Download Pojav Launcher](https://play.google.com/store/apps/details?id=net.kdt.pojavlaunch)
   * Do not fall into scam websites and apps. The only official site for it is https://github.com/PojavLauncherTeam/PojavLauncher

> 2. [Download Material Files](https://play.google.com/store/apps/details?id=me.zhanghai.android.files)

Author: Might want to rewrite the instructions with the system app, because that ensures it works without root and may even have easier access to the .minecraft folder
https://play.google.com/store/apps/details?id=com.marc.files

   * This is the file manager we'll be using for this tutorial. You could use others too, but instructions may vary.
3. [Download Fabric installer](https://fabricmc.net/use/installer/) (jar version)
4. [Download the latest **MultiMC version** of Fabulously Optimized](https://www.curseforge.com/minecraft/modpacks/fabulously-optimized/files)
   * Don't see the MultiMC version? Click the title of the CurseForge version you need and scroll down, you'll find the MultiMC variant there.
      * Alternatively, look for it in the sidebar, under "server packs". Don't worry, these are not server packs, just marked as such for findability.
   * If you don't see any mods inside the zip or only see one, you downloaded the wrong version.

> 5. [Download Fabulously Optimized patches for Pojav Launcher](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/Fabulously-Optimized/fabulously-optimized/tree/pojav/PojavLauncher/1.19.2)

Author: todo: 1.19.3 folder and maybe multiple links here?

   * This optimizes the FO experience for mobile users.
6. Run Pojav Launcher
7. Tap `Add account` → `Microsoft account` and sign in
8. Tap the settings button → `Video and renderer` → `ANGLE (release 1.17+, mid)` → `Back to the last screen`→ `Back to the last screen`
9. Tap `Create new profile` → `Create new profile` → `Save profile`
10. Tap `Play`. Wait for it to fully load to the title screen, then exit the game.
    * If you can't get past a black screen, try creating a new 1.18.2 profile first - it seems to download the necessary Java and dependencies.
11. Launch Pojav again, tap `Install .jar` → select `fabric-installer-x.x.x.jar`. You'll see the Fabric installer with some logs.
12. Tap `❌` to close the logs, then `Install` → `OK` → `⛝`
13. Close Pojav and open Material Files.
14. Navigate to `Download` → `Fabulously+Optimized+x.x.x.zip` → `Fabulously Optimized x.x.x` → `minecraft`
15. Hold down on one folder and tap on all three folders, then `Extract` button at the top

> 16. Navigate back to the root directory, then `Android` → `data` → `net.kdt.pojavlaunch` → `files` → `.minecraft`

Author: Android <9 path is
`/storage/emulated/0/games/PojavLauncher/.minecraft`

2. [Download Material Files](https://play.google.com/store/apps/details?id=me.zhanghai.android.files)
   * This is the file manager we'll be using for this tutorial. You could use others too, but instructions may vary.
3. [Download Fabric installer](https://fabricmc.net/use/installer/) (jar version)
4. [Download the latest **MultiMC version** of Fabulously Optimized](https://www.curseforge.com/minecraft/modpacks/fabulously-optimized/files)
   * Don't see the MultiMC version? Click the title of the CurseForge version you need and scroll down, you'll find the MultiMC variant there.
      * Alternatively, look for it in the sidebar, under "server packs". Don't worry, these are not server packs, just marked as such for findability.
   * If you don't see any mods inside the zip or only see one, you downloaded the wrong version.
5. [Download Fabulously Optimized patches for Pojav Launcher](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/Fabulously-Optimized/fabulously-optimized/tree/pojav/PojavLauncher/1.19.2)
   * This optimizes the FO experience for mobile users.
6. Run Pojav Launcher
7. Tap `Add account` → `Microsoft account` and sign in

> 8. Tap the settings button → `Video and renderer` → `ANGLE (release 1.17+, mid)` → `Back to the last screen`→ `Back to the last screen`

Author: "remove it for 2x more fps"

as indeed, it doesn't seem to be required now 
but should test first
